Communication guide: overview, blogs, podcast, social media, presentation, lay summary chapters (#1929)
alan-turing-institute/the-turing-way: book/website/communication/presentations/presentations-personal-stories.md

Both in her role as a researcher and within the R community, she works to make science as reproducible and open as possible using open informatics tools.

## 1. When did you start giving talks about your research in non-academic events?

I work at [INTA](https://www.argentina.gob.ar/inta) (National Institute of Agricultural Technology), which is a scientific institution.
Our activities are organized into research & development and knowledge transfer & extension, both with the same weight and importance.
Everything that has to do with teaching, outreach, communicating, and transferring knowledge of our production to social actors in the agricultural sector and the general public is part of our work.
Within INTA, training and communication activities, whether through a course, a workshop, an article, or an interview in a newspaper, radio, or television, are important and necessary.

> We only make sense if people use what we do.

Reviewer: This sentence isn't entirely clear to me: We only make sense or our activities only make sense?

## 2. What aspects do you have to take into account when thinking about an event with very interdisciplinary talks?

I think the main thing is to know your audience beforehand.

> You can ask the organizers of the event to whom you are going to speak, what is their background, what is the reason they come to the event, and so on., since, in that way, you can foresee some of the expectations they are going to have and try, at least, to fulfil some of them in your talk.

Reviewer: ```suggestion
You can ask the organizers of the event to whom you are going to speak, what is their background, what is the reason they come to the event, and so on. 
In that way, you can foresee some of the expectations they are going to have and try to fulfill some of them in your talk.
```

Also, I start the talk with a relaxed attitude so that the audience feels that, even though I am there as an expert,
I am approachable when it comes to comments and questions.
After all, these kinds of meetings and sessions are meant for people to take something away, to learn something.

> Knowing your audience and at least one or two of the things they expect from the talk is important because, by fulfilling them, they leave with something they went looking for and so it's going to contribute to the evaluation that it was worth it being there.

Reviewer: ```suggestion
Knowing your audience and at least one or two of the things they expect from the talk is important. 
By fulfilling these expectations the audience leaves with a sense that it was worth being there.
```

That is another point to take into account, respect for the other person's time and to prepare yourself in advance.

> Make it clear that you care about giving a good talk, with useful information, make it clear that you want them to profit from the time they decided to spend listening to you.

Reviewer: ```suggestion
Make it clear that you care about giving a good talk, with useful information. 
Tell your audience that you want them to profit from the time they decided to spend listening to you.
```

It is necessary to be well informed about the format of the talk: the amount of time allotted to the talk, and its structure (like you deliver your talk and questions are asked at the end or you can ask questions during the talk, how much time is given to questions, are debates and discussions expected).

In general, if there is no predefined format for the talk, I ask to be allowed to interact with the audience.
I like to ask questions and have them answer me in the chat, it makes them part of the event.
Then, I read the answers and, even though the slide is the same, what I am going to say is going to be different depending on the answers I get.

If the format is structured, I follow the instructions of the organization, which in general consists of an introduction of the speaker, the presentation, and then the questions.
Some organizers ask for the material beforehand so that the audience can go through it and send you the questions before the talk.
So you need to communicate with the organizers and understand the process.
Also, I always ask for feedback from the organizers so that I can improve the talk.

> On the other hand, I try to address what was left unanswered at the event in two different ways: I make a blog post with the questions and answers (sometimes I add them on the talk’s page) or with Twitter threads.

Reviewer: ```suggestion
I also try to address what was left unanswered at the event in two different ways: I make a blog post with the questions and answers (sometimes I add them on the talk’s page) or with Twitter threads.
```

In the same way, I strive to make the materials available before the talk and take into account accessibility issues, such as a font size that is visible, colour contrasts, including speaker's notes, a download link, a license so they know how to use it, among other details.

> Thanks to R-Ladies I started to pay more attention to these kinds of details in my materials, things that I didn't pay much attention to before but can make a big difference in the use and accessibility of my talks.

Reviewer: ```suggestion
Thanks to [R-Ladies](https://rladies.org/) I started to pay more attention to these kinds of details in my materials, things that I didn't pay much attention to before but can make a big difference in the use and accessibility of my talks.
```

## 3. What aspects of delivering these talks contribute to your professional work?

The foremost aspect is the network of contacts you gain from the events.
By attending and talking about what I’m working on has resulted in collaborations, agreements, publications, joint work, and projects.

Expanding that network of contacts can also mean that you can bring people together to work jointly.
I think it is a very important role for those of us who have been in the profession for several years to be a bridge between communities and people so that they can get to know each other.

> On the other hand, it helps me to improve my talks, my materials (many of them end up in a somewhat more academic publication), and to gauge the needs, doubts, or concerns of people regarding the topic I am communicating.

Reviewer: ```suggestion
Delivering talks also helps me to improve my talks, my materials (many of them end up in a somewhat more academic publication), and to gauge the needs, doubts, or concerns of people regarding the topic I am communicating.
```

> That serves me as a thermometer of demand, of what people are needing; of what is not being understood; of what topics require training, books, Blogposts, or materials; to reaffirm or correct the direction of my work.

Reviewer: ```suggestion
That serves me as a thermometer of demand, of what people are needing; of what is not being understood; of what topics require training, books, blogposts, or materials; to reaffirm or correct the direction of my work.
```

When I prepare a talk, when budgeting the time it takes me to put it together, I add a day or two more to check its accessibility.
The license is fundamental to let your audience know how they can reuse your material and, if they cannot reuse it, to make it clear.

Finally, a very Argentine phrase: "Don’t get up on your high horse"; deliver the talk with the peace of mind that comes from knowing your work and your discipline, but also with fitting humility.
You have to acknowledge that there may be questions that you do not know how to answer.

The same goes for language, where you can be technically correct without using complex language.
However, if you use technical terms because you have to, you have to explain them or make an analogy with everyday things.
For example, a computer program is like a recipe: you give instructions to the computer.
That is why it is key to know your audience, as then you know what to rely on when explaining more complex concepts.

We are discussing talks given at community events.

> In general, I deliver them free of charge and the materials are open, but if a company invites you, don't be ashamed or afraid to charge for them in case that, later on, it will be economically profitable for whoever invites you.

Reviewer: ```suggestion
In general, I deliver them free of charge and the materials are open, but if a company invites you, don't be ashamed or afraid to charge them for your contribution.
```

I also don't pay out of my pocket if I am invited to give a talk; this means that the costs of travel or registration are paid by the organizer.
If your talks are in high demand, the organizer's handling of the costs helps you decide.

> Also, you should never transfer the intellectual property of your material to the event organizer, but if they ask you to transfer it and you agree, charge them very well for it.

Reviewer: ```suggestion
Also, you should never transfer the intellectual property of your material to the event organizer. 
If organizers ask you to transfer your rights you should charge them for it.
```

If I am invited to deliver a talk as an INTA researcher, I do not charge for it either, because it is part of my job.
When I am invited to deliver a talk due to my involvement in communities of practice or for events not related to INTA, if they are for public universities, for example, organizations like R-Ladies, or free events and conferences, I accept as a way of giving back a big part of what I received.
I live in a country that allowed me to receive education in a public system that is funded by all of us, and giving this kind of training is a way of giving back.
Currently, I’m privileged enough to be able to engage in this kind of activity without charging; not everyone is, and they don't have to feel obligated to do it.

Some people have a guideline when it comes to being invited to deliver a talk, such as [Tatiana Mac](https://gist.github.com/tatianamac/493ca668ee7f7c07a5b282f6d9132552#file-tatiana-mac-speaker-rider-md), [Greg Wilson](https://third-bit.com/talks/speaking/) and Steph Locke.

These are also questions that I take into account: Is it an all-male panel?

> Does the event have a Code of Conduct?

Reviewer: Do we have a code of conduct section to link to?
